### PR TITLE
Bug 1451910 - Caution about egress network policy limitation

### PR DESCRIPTION
**Pull request by pravisankar**

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/team-documentation PTAL

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -258,6 +258,14 @@ take up to one minute to adapt to these changes.
 # oc create -f <policy>.json
 ----
 
+[CAUTION]
+====
+Exposing services by creating xref:../dev_guide/routes.adoc#creating-routes[*routes*]
+will ignore `EgressNetworkPolicy`. We do egress network policy service endpoint filtering
```
Comment (ahardin-rh) [inline admin_guide/managing_networking.adoc]: s/*routes*/routes

Comment (ahardin-rh) [inline admin_guide/managing_networking.adoc]: Egress network policy service endpoint filtering is done at node `kubeproxy`. When the router is involved, `kubeproxy` is bypassed and egress network policy enforcement is not applied.

File: admin_guide/managing_networking.adoc [hunk 2]
```diff
@@ -258,6 +258,14 @@ take up to one minute to adapt to these changes.
...
+at node kubeproxy and when router is involved, kubeproxy is bypassed and egress
```
Comment (knobunc) [inline admin_guide/managing_networking.adoc]: at _the_ node kubeproxy ... when _the_ router is involved, ...

**Review by knobunc — CHANGES_REQUESTED** [review]
Do we want to suggest that if admins want to prevent this bypass, they can limit the ability of people to create routes?

**Comment by pravisankar** [comment]
@knobunc @ahardin-rh Updated, ptal

**Review by knobunc — APPROVED** [review]
LGTM, thanks Ravi

**Comment by ahardin-rh** [comment]
Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/managing_networking.adoc#admin-guide-manage-networking[Managing Networking]
|Added a note about limitations with the egress network policy.
%

**Comment by ahardin-rh** [comment]
@knobunc Is this for OCP 3.6 forward, or can it apply to other versions?

**Comment by pravisankar** [comment]
@ahardin-rh Applies to OCP 3.3+ (starting from version where we introduced egress network policy)

**Pull request merged**